var/spack/repos/builtin/packages/py-pandas/package.py

```diff
@@ -38,6 +38,8 @@ class PyPandas(PythonPackage):
     homepage = "http://pandas.pydata.org/"
     url = "https://pypi.io/packages/source/p/pandas/pandas-0.19.0.tar.gz"
```

Comment:
Don't we typically keep the URL up-to-date with the most recent version?

Reply:
Per other commits and PRs it doesn't appear to be the case, though I'm more than happy to.  There didn't seem to be anything in the contribution guide regarding this.